---
layout: page
title: Publications and Preprints
permalink: /publications/
---
For publications with more than a handful of authors, only the first
few authors are listed.   
A list of publications is also available through
[Google Scholar](https://scholar.google.com/citations?hl=en&user=jMaIpR4AAAAJ&view_op=list_works&sortby=pubdate)
<br>
<br>

\* _indicates equal contribution_  
&#8224; _indicates co-corresponding author_  <br>
&amp; _indicates authors are listed alphabetically_

<br>
<br>

__preprints__
* Aguirre, M.&#8224;, __Spence, J. P.__&#8224;, Sella, G.&#8224;, and Pritchard, J. K.&#8224; [Regulatory network topology and the genetic architecture of gene expression.](https://doi.org/10.1101/2025.08.12.669924) _bioRxiv preprint_.

* Ota, M.&#8224;, __Spence, J. P.__, Zeng, T., Dann, E., Marson, A.&#8224;, and Pritchard, J. K.&#8224; [Causal modeling of gene effects from regulators to programs to traits: integration of genetic associations and Perturb-seq.](https://doi.org/10.1101/2025.01.22.634424) _bioRxiv preprint_.

* __Spence, J. P.__\*&#8224;, Mostafavi, H.\*&#8224;, Ota, M.&#8224;, Milind, N., Gjorgjieva, T., Smith, C. J., Simons, Y. B., Sella, G., and Pritchard, J. K.&#8224; [Specificity, length, and luck: How genes are prioritized by rare and common variant association studies.](https://www.biorxiv.org/content/10.1101/2024.12.12.628073v1) _bioRxiv preprint_.
    [\[.pdf\]](/assets/specificity.pdf/)

* Milind, N.&#8224;, Smith, C. J., Zhu, H., __Spence, J. P.__&#8224;, and Pritchard, J. K.&#8224; [Buffering and non-monotonic behavior of gene dosage response curves for human complex traits.](https://doi.org/10.1101/2024.11.11.24317065) _medRxiv preprint_.

* Mualim, K. S.\*, __Spence, J. P.__\*, Weiß, C. L.\*, Selmoni, O., Lin, M., and Expósito-Alonso, M. [Genetic diversity loss in the Anthropocene will continue long after habitat destruction ends.](https://doi.org/10.1101/2024.10.21.619096) _bioRxiv preprint_.


__2025__
* Judd, J., __Spence, J. P.__, Pritchard, J. K., Kachuri, L., and Witte, J. S. [Investigating the role of neighborhood socioeconomic status and germline genetics on prostate cancer risk.](https://doi.org/10.1016/j.xhgg.2025.100492) _HGG Advances_ (2025).


* Aguirre, M.&#8224;, __Spence, J. P.__&#8224;, Sella, G.&#8224;, and Pritchard, J. K.&#8224; [Gene regulatory network structure informs the distribution of perturbation effects.](https://doi.org/10.1371/journal.pcbi.1013387) _PLoS Computational Biology_ (2025).

* Schraiber, J. G., __Spence, J. P.__, and Edge, M. D. [Estimation of demography and mutation rates from one million haploid genomes.](https://doi.org/10.1016/j.ajhg.2025.07.008) _American Journal of Human Genetics_ (2025).


* Smith, C. J.&#8224;, Strausz, S.&#8224;, FinnGen, __Spence, J. P.__&#8224;, Ollila, H. M.&#8224;, and Pritchard, J. K.&#8224; [Haplotype analysis reveals pleiotropic disease associations in the HLA region.](https://doi.org/10.1016/j.ajhg.2025.06.011) _American Journal of Human Genetics_ (2025).

* Batra, S. S.\*, Cabrera, A.\*, __Spence, J. P.__\*, Goell, H., Anand, S. S., Hilton, I. B., and Song, Y. S. [Predicting the effects of CRISPR-Cas9-based epigenome editing.](https://doi.org/10.7554/eLife.92991) _eLife_ (2025).


__2024__
* Patel, R. A.&#8224;, Weiß, C. L., Zhu, H., Mostafavi, H., Simons, Y. B., __Spence, J. P.__&#8224;, and Pritchard, J. K.&#8224; [Characterizing selection on complex traits through conditional frequency spectra.](https://doi.org/10.1093/genetics/iyae210) _Genetics_ (2024).


*  Czech, L. __Spence, J. P.__, and Expósito-Alonso, M. [grenedalf: population genetic statistics for the next generation of pool sequencing.](https://doi.org/10.1093/bioinformatics/btae508) _Bioinformatics_ (2024).

*  Rothschild, D.\*, Susanto, T. T.\*, Sui, X.\*, __Spence, J. P.__, Rangan, R., Genuth, N. R., Sinnott-Armstrong, N., Wang, X., Pritchard, J. K.&#8224;, and Barna, M.&#8224; [Diversity of ribosomes at the level of rRNA variation associated with human health and disease.](https://doi.org/10.1016/j.xgen.2024.100629) _Cell Genomics_ (2024).
    [\[.pdf\]](/assets/ribosomes.pdf/)
    [\[supplement\]](/assets/ribosomes_supp.pdf)
    [\[.ris\]](/assets/ribosomes.ris)


*  Zeng, T.\* <sup>,</sup> &#8224;, __Spence, J. P.__\* <sup>,</sup> &#8224;, Mostafavi, H., and Pritchard, J. K.&#8224; [Bayesian estimation of gene constraint from an evolutionary model with gene features.](https://doi.org/10.1038/s41588-024-01820-9) _Nature Genetics_ (2024).
    [\[.pdf\]](/assets/genebayes.pdf)
    [\[supplement\]](/assets/genebayes_supp.pdf)
    [\[.ris\]](/assets/genebayes.ris)


*  Antonio, M. L.\*, Weiß, C. L.\*, Gao, Z.\*, Sawyer, S.\*, Oberreiter, V, Moots, H. M., __Spence, J. P.__, _et al._ [Stable population structure in Europe since the Iron Age, despite high mobility.](https://doi.org/10.7554/eLife.79714.sa0) _eLife_ (2024).
    [\[.pdf\]](/assets/atqy.pdf)
    [\[.ris\]](/assets/atqy.ris)


*  Aw, A. J., __Spence, J. P.__, and Song, Y. S.
[A simple and flexible test of sample exchangeability with applications to statistical genomics.](https://doi.org/10.1214/23-AOAS1817) _Annals of Applied Statistics_ (2024).
    [\[.pdf\]](/assets/exchangeability.pdf)
    [\[supplement\]](/assets/exchangeability_supp.pdf)
    [\[.ris\]](/assets/exhangeability.ris)



__2023__
*  Mostafavi, H.&#8224;, __Spence, J. P.__, Naqvi, S., and Pritchard, J. K.&#8224;.  [Systematic differences in discovery of genetic effects on gene expression and complex traits.](https://doi.org/10.1038/s41588-023-01529-1) _Nature Genetics_ (2023).
    [\[.pdf\]](/assets/gwas_eqtl.pdf)
    [\[supplement\]](/assets/gwas_eqtl_supp.pdf)
    [\[.ris\]](/assets/gwas_eqtl.ris)

* __Spence, J. P.__&#8224;, Zeng, T., Mostafavi, H., and Pritchard, J. K. [Scaling the Discrete-time Wright-Fisher model to biobank-scale datasets.]( https://doi.org/10.1093/genetics/iyad168) _Genetics_ (2023).
    [\[.pdf\]](/assets/dtwf.pdf)
    [\[.ris\]](/assets/dtwf.ris)

*  Moots, H. M., Antonio, M. L.\*, Sawyer, S.\*, __Spence, J. P.__\*, Oberreiter, V.\*, Weiß, C. L.\*, Lucci, M.\*, Cherifi, Y. M. S.\*, _et al._ [A genetic history of continuity and mobility in the Iron Age central Mediterranean.](https://doi.org/10.1038/s41559-023-02143-4) _Nature Ecology & Evolution_, Vol. 7 (2023) 1515-1524.
     [\[.pdf\]](/assets/iron_age_med.pdf)
     [\[.ris\]](/assets/iron_age_med.ris)


*  Schreiber, J.\*, Boix, C.\*, _et al._ [The ENCODE Imputation Challenge: A critical assessment of methods for cross-cell type imputation of epigenomic profiles.](https://doi.org/10.1186/s13059-023-02915-y) _Genome Biology_, Vol. 24 (2023).
    [\[.pdf\]](/assets/encode_imputation_challenge.pdf)
    [\[.ris\]](/assets/encode_imputation_challenge.ris)

* Fan, S.\*, __Spence, J. P.__\*, Feng, Y., Hansen, M. E. B., Terhorst, J., Beltrame, M. H., Ranciaro, A., Hirbo, J., Beggs, W., Thomas, N., Nyambo, T., Mpoloka, S. W., Mokone, G. G., Njamnshi, A., Folkunang, C., Meskel, D. W., Belay, G., Song, Y. S., and Tishkoff, S. A. [Whole-genome sequencing reveals a complex African population demographic history and signatures of local adaptation.](https://doi.org/10.1016/j.cell.2023.01.042) _Cell_, Vol. 186, Issue 5 (2023).
    [\[.pdf\]](/assets/whole_genome_sequencing.pdf)
    [\[.ris\]](/assets/whole_genome_sequencing.ris)

*  Lauterbur, M. E., _et al._ [Expanding the stdpopsim catalog, and lessons learned for realistic genome simulations.](https://doi.org/10.7554/eLife.84874.1) _eLife_ reviewed preprint (2023).


__2022__
*  __Spence, J. P.__&#8224;, Sinnott-Armstrong, N., Assimes, T., and Pritchard, J. K.&#8224; [A flexible modeling and inference framework for estimating variant effect sizes from GWAS summary statistics.](https://doi.org/10.1101/2022.04.18.488696) _bioRxiv_.

*  Czech, L.\*, Peng, Y.\*, __Spence, J. P.__, _et al._ [Monitoring rapid evolution of plant populations at scale with Pool-sequencing.](https://doi.org/10.1101/2022.02.02.477408) _bioRxiv_.

*   Expósito-Alonso, M., Booker, T. R.<sup>&amp;</sup>, Czech, L.<sup>&amp;</sup>, Gillespie, L.<sup>&amp;</sup>, Hately, S.<sup>&amp;</sup>, Kyriazis, C. C.<sup>&amp;</sup>, Lang, P. L. M.<sup>&amp;</sup>, Leventhal, L.<sup>&amp;</sup>, Nogues-Bravo, D.<sup>&amp;</sup>, Pagowski, V.<sup>&amp;</sup>, Ruffley, M.<sup>&amp;</sup>, __Spence, J. P.__<sup>&amp;</sup>, Toro Arana, S. E.<sup>&amp;</sup>, Weiß, C. L.<sup>&amp;</sup>, and Zess, E.<sup>&amp;</sup>  [Genetic diversity loss in the Anthropocene.](https://doi.org/10.1126/science.abn5642) _Science_, Vol. 377, Issue 6613 (2022).
    [\[.pdf\]](/assets/genetic_diversity_loss.pdf)
    [\[supplement\]](/assets/genetic_diversity_loss_supp.pdf)
    [\[.ris\]](/assets/genetic_diversity_loss.ris)


*   Patel, R. A.\*, Musharoff, S. A.\*, __Spence, J. P.__, _et al._
   [Genetic interactions drive heterogeneity in causal variant effect sizes for gene expression and complex traits.](https://doi.org/10.1016/j.ajhg.2022.05.014) _American Journal of Human Genetics_ (2022).
    [\[.pdf\]](/assets/genetic_interactions.pdf)
    [\[supplement\]](/assets/genetic_interactions_supp.pdf)
    [\[.ris\]](/assets/genetic_interactions.ris)


__2021__
*   Naqvi, S.\*, Sleyp, Y.\*, Hoskens, H., Indencleef, K., __Spence, J. P.__, _et al._
    [Shared heritability of human face and brain shape.](https://doi.org/10.1038/s41588-021-00827-w) _Nature Genetics_ (2021).
    [\[.pdf\]](/assets/shared_heritability_of_human_face_and_brain_shape.pdf)
    [\[supplement\]](/assets/shared_heritability_of_human_face_and_brain_shape_supp.pdf)
    [\[.ris\]](/assets/shared_heritability_of_human_face_and_brain_shape.ris)


__2020__
*   __Spence, J. P.__ [Flexible mean field variational inference using mixtures of
    non-overlapping exponential families.](https://proceedings.neurips.cc/paper/2020/hash/e3a54649aeec04cf1c13907bc6c5c8aa-Abstract.html)
    Advances in Neural Information Processing Systems 33 (NeurIPS 2020).
    [\[.pdf\]](/assets/non_overlapping_mixtures.pdf)
    [\[supplement\]](/assets/non_overlapping_mixtures_supp.pdf)
    [\[.ris\]](/assets/non_overlapping_mixtures.ris)


__2019__
*   __Spence, J. P.__&#8224; and Song, Y. S.&#8224; 
    [Inference and analysis of population-specific fine-scale recombination 
    maps across 26 diverse human populations.](https://doi.org/10.1126/sciadv.aaw9206)
    _Science Advances_, Vol. 5, No. 10, eaaw9206 (2019).
    [\[.pdf\]](/assets/inference_and_analysis_pyrho.pdf)
    [\[supplement\]](/assets/inference_and_analysis_pyrho_supp.pdf)
    [\[.ris\]](/assets/inference_and_analysis_pyrho.ris)
    [\[method: pyrho\]](https://github.com/popgenmethods/pyrho/)

*   Steinr&uuml;cken, M., Kamm, J., __Spence, J. P.__, and Song, Y. S.
    [Inference of complex population histories using whole-genome sequences
    from multiple populations.](https://doi.org/10.1073/pnas.1905060116)
    _PNAS_, Vol. 116, No. 34 (2019) 17115-17120.
    [\[.pdf\]](/assets/dical2.pdf)
    [\[supplement\]](/assets/dical2_supp.pdf)
    [\[.ris\]](/assets/dical2.ris)
    [\[method: dical2\]](https://sourceforge.net/projects/dical2/)


__2018__ 
*   Chan, J., Perrone, V., __Spence, J. P.__, Jenkins, P., Mathieson, S., and Song Y. S.
    [A likelihood-free inference framework for population genetic data using
    exchangeable neural networks.](http://papers.nips.cc/paper/8078-a-likelihood-free-inference-framework-for-population-genetic-data-using-exchangeable-neural-networks)
    Advances in Neural Information Processing Systems 31 (NeurIPS 2018).
   [\[.pdf\]](/assets/exchnn.pdf)
   [\[supplement\]](/assets/exchnn_supp.pdf)
   [\[.ris\]](/assets/exchnn.ris)

*   Moreno-Mayar, J. V.\*, Vinner, L.\*, de Barros Damgaard, P.\*, de la Fuente C.\*,
    Chan, J.\*, __Spence, J. P.__\*, _et al._ 
    [Early human dispersals within the 
    Americas.](https://doi.org/10.1126/science.aav2621)
    _Science_, Vol. 362, Issue 6419 (2018).
    [\[.pdf\]](/assets/early_human_dispersal_americas.pdf)
    [\[supplement\]](/assets/early_human_dispersal_americas_supp.pdf)
    [\[.ris\]](/assets/early_human_dispersal_americas.ris)

*   Steinr&uuml;cken, M., __Spence, J. P.__, Kamm, J. A., Wieczorek, E.
    and Song, Y. S. 
    [Model-based detection and analysis of introgressed Neanderthal
    ancestry in modern humans.](https://doi.org/10.1111/mec.14565)
    _Molecular Ecology_, Vol. 27, No. 19 (2018) 3873-3888.
    [\[.pdf\]](/assets/dical_admix.pdf)
    [\[supplement\]](/assets/dical_admix_supp.pdf)
    [\[.ris\]](/assets/dical_admix.ris)

*   __Spence, J. P.__, Steinr&uuml;cken, M., Terhorst, J., and Song, Y. S.
    [Inference of population history using coalescent HMMs:
    review and outlook.](https://doi.org/10.1016/j.gde.2018.07.002)
    _Current Opinion in Genetics & Development_, Vol. 53 (2018) 70-76.
   [\[.pdf\]](/assets/hmm_review.pdf)
   [\[.ris\]](/assets/hmm_review.ris)


__2016__
*   Mallick _et al._
    [The Simons Genome Diversity Project: 300 genomes from 142 diverse
    populations.](http://dx.doi.org/10.1038/nature18964)
    _Nature_, Vol. 538 (2016) 201-206.
    [\[.pdf\]](/assets/sgdp.pdf)
    [\[supplement\]](/assets/sgdp_supp.pdf)
    [\[.ris\]](/assets/sgdp.ris)

*  Kamm, J. A.\*, __Spence, J. P.__\*, Chan, J., and Song, Y. S.
    [Two-locus likelihoods under variable population size and fine-scale
    recombination rate estimation.](http://dx.doi.org/10.1534/genetics.115.184820)
    _Genetics_, Vol. 203, No. 3 (2016) 1381-1399.
    [\[.pdf\]](/assets/ldpop.pdf)
    [\[.ris\]](/assets/ldpop.ris)

*  __Spence, J. P.__\*, Kamm, J. A.\*, and Song, Y. S.
    [The site frequency spectrum for general
    coalescents.](http://dx.doi.org/10.1534/genetics.115.184101)
    _Genetics_, Vol. 202, No. 4 (2016) 1549-1561.
    [\[.pdf\]](/assets/lambda.pdf)
    [\[.ris\]](/assets/lambda.ris)
